relayer/chains/cosmos/tx.go

```diff
@@ -1305,9 +1358,22 @@ func (cc *CosmosProvider) QueryICQWithProof(ctx context.Context, path string, re
 	if err != nil {
 		return provider.ICQProof{}, fmt.Errorf("failed to execute interchain query: %w", err)
 	}
+
+	proofOps := &crypto.ProofOps{}
```

Comment:
Is this needed anymore? If so, as a method within the relayer `client` package would be good. This can also start as a nil slice to be consistent `var proofOps crypto.ProofOps`

Reply:
Nope, we no longer need any of the explicit proof ops conversions in the provider implementations since the client wrapper does this now.